test/test_compressed_support.py

```diff
@@ -0,0 +1,97 @@
+#-------------------------------------------------------------------------------
+# elftools tests
+#
+# Pierre-Marie de Rodat (pmderodat@kawie.fr)
+# This code is in the public domain
```

Comment:
I think it’s possible, as though you need to have a fairly recent readelf version: it looks like this feature has been added in readelf in May 2015: https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git;a=commit;f=binutils/readelf.c;h=0e602686df5677fee06cbd1718b4a7aa5379cd2a